src/sources/kubernetes_logs/pod_metadata_annotator.rs

```diff
@@ -436,6 +451,32 @@ fn annotate_from_container_status(
         )
     }
 
+    match &container_status.state {
+        Some(ContainerState {
+            running:
+                Some(ContainerStateRunning {
+                    started_at: Some(start_time),
+                }),
+            ..
+        }) => {
+            let legacy_key = fields_spec
+                .container_start_time
+                .path
+                .as_ref()
+                .map(|k| &k.path)
+                .map(LegacyKey::Overwrite);
+
+            log_namespace.insert_source_metadata(
+                Config::NAME,
+                log,
+                legacy_key,
+                path!("container_start_time"),
+                start_time.0.to_string(),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏these changes look correct to me but I believe the unit test `test_annote_from_container_status()` will need to be updated accordingly.

Reply:
added!